application/helpers/admin/export/SurveyObj.php

```diff
@@ -366,9 +379,39 @@ public function getFullAnswer($fieldName, $answerCode, Translator $translator, $
     }
 
     /**
+    * Returns the short answer for the question.
+    *
+    * @param string $sFieldName
+    * @param string $sValue
+    * @return string
+    */
+    public function getShortAnswer($sFieldName, $sValue) {
+        $aQuestion = $this->fieldMap[$sFieldName];
+        $sFieldType = $aQuestion['type'];
+
+        switch ($sFieldType)
+        {
+            case 'K':
+            case 'N':
+                if (trim($sValue)!='') {
+                    if (strpos($sValue,".")!==false) {
+                        $sValue=rtrim(rtrim($sValue,"0"),".");
+                    }
+                    $aQidAttributes = getQuestionAttributeValues($aQuestion['qid']);
+                    if (isset($aQidAttributes['num_value_int_only']) && $aQidAttributes['num_value_int_only']) {
```

Comment:
Just a remark here : num_value_int_only can be set after survey activation. Then some user can have float value with num_value_int_only .
If the value is an int : double rtrim do the job. Force to be int can remove some information.